extensions/resource-deployment/src/ui/deployClusterWizard/pages/clusterSettingsPage.ts

```diff
@@ -83,6 +83,41 @@ export class ClusterSettingsPage extends WizardPageBase<DeployClusterWizard> {
 			]
 		};
 
+		const dockerSection: SectionInfo = {
+			labelPosition: LabelPosition.Left,
+			collapsed: true,
+			collapsible: true,
+			title: localize('deployCluster.DockerSettings', "Docker settings"),
+			fields: [
+				{
+					type: FieldType.Text,
+					label: localize('deployCluster.DockerRegistry', "Registry"),
+					required: true,
+					variableName: VariableNames.DockerRegistry_VariableName
+				}, {
+					type: FieldType.Text,
+					label: localize('deployCluster.DockerRepository', "Repository"),
+					required: true,
+					variableName: VariableNames.DockerRepository_VariableName
+				}, {
+					type: FieldType.Text,
+					label: localize('deployCluster.DockerImageTag', "Image tag"),
+					required: true,
+					variableName: VariableNames.DockerImageTag_VariableName
+				}, {
+					type: FieldType.Text,
+					label: localize('deployCluster.DockerUsername', "Username"),
+					required: false,
```

Comment:
docker username is not required? I remember providing it on the notebook.

Reply:
it is not strictly required, for public registries, it is not.